src/components/ar-paint-controls.js

```diff
@@ -0,0 +1,155 @@
+AFRAME.registerSystem('ar-paint-controls', {
+  numberStrokes: 0
+});
+
+/* globals AFRAME THREE */
+AFRAME.registerComponent('ar-paint-controls', {
+  dependencies: ['brush', 'raycaster'],
+
+  init: function () {
+    var el = this.el;
+    this.controller = null;
+
+    this.size = el.sceneEl.renderer.getSize();
+    this.pointer = new THREE.Vector2();
+    // normalized device coordinates position
+    this.pointerNdc = new THREE.Vector2();
```

Comment:
I'd go for a full name even if long, but make it more understandable without the comment, something like `normalizedCoordinatedPositionPointer` or so. Is not used widely anyway like to be a problem